plugins/processors/reversedns/parallel/unordered.go

```diff
@@ -0,0 +1,58 @@
+package parallel
+
+import (
+	"context"
+	"sync"
+
+	"github.com/influxdata/telegraf"
+	"golang.org/x/sync/semaphore"
+)
+
+type Unordered struct {
+	wg    sync.WaitGroup
+	acc   telegraf.Accumulator
+	queue chan telegraf.Metric
+	sem   *semaphore.Weighted
```

Comment:
Why is this a weighted semaphore?  It's only ever acquired/released with weight 1.

Reply:
I don't think there is un unweighted implementation, since it's the same as this^ 
https://www.godoc.org/golang.org/x/sync/semaphore

Reply:
Ok.  I guess I expected there would be a simple api where you don't have to specify weight every time.  Thanks for the info.